### PR TITLE
Major Update and Simplification

### PR DESCRIPTION
**Pull request by aaronedell**
Combined the CID, KPI, and CUDOS deployment options. Made everything simpler and easier to follow. Added more FAQs.

1. Combined deployment methods into a single page. 
2. Made it clear that CID CMD is the way to do things first. 
3. Clarified other places people were getting stuck.
4. Added more FAQs.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1_prerequistes.md [hunk 1]
```diff
@@ -508,6 +508,8 @@ Before you can use the AWS CloudFormation template to automate an Athena integra
 
 To streamline and automate integration of your Cost and Usage Reports with Athena, AWS provides an AWS CloudFormation template with several key resources along with the reports you setup for Athena integration. The AWS CloudFormation template includes an AWS Glue crawler, an AWS Glue database, and an AWS Lambda event.
 
+If you are not deploying the CIDs in your payer acacount, or wish to deploy them on top of multiple payer accounts, please follow [these instructions](https://wellarchitectedlabs.com/cost/200_labs/200_cloud_intelligence/faq/) in lieu of the below. Come back for the QuickSight prerequisites.
+
```
Comment (iakov-aws) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1_prerequistes.md:512]: For multi payer we have this pr #655 so i suggest to skip it here

File: content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md [hunk 2]
```diff
@@ -142,6 +142,33 @@ These actions should be done in Governance account
 
 {{% /expand%}}
 
+## I'm getting an error in QuickSight that is saying Athena timed out?
+
+For very large CUR files, Athena may time out trying to query the data for summary_view. In Athena, find the summary_view view, click the three dots next to it and select show/edit query. Modify the following:
+
+- Adjust the granularity to monthly, by changing 'day' to 'month' in row 6 
+- Or, adjust the look back from '7' months to desired time-frame in row 75
+- In QuickSight, refresh your dataset.
```
Comment (iakov-aws) [inline content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md:151]: Request limit increase?

**Review by iakov-aws — APPROVED** [review]

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md [hunk 3]
```diff
@@ -0,0 +1,1233 @@
...
+
+15.	Click **Cancel** to exit
+
+    ![qs_cancel_refresh](/Cost/200_Cloud_Intelligence/Images/cur/qs_cancel_refresh.png?classes=lab_picture_small)
+
+16.	Click **x** to exit
+
+    ![qs_exit_refresh](/Cost/200_Cloud_Intelligence/Images/cur/qs_exit_refresh.png?classes=lab_picture_small)
+
+17.	Repeat these steps with the KPI datasets. No refresh is required for customer_all.
```
Comment (iakov-aws) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:77]: Repeat these steps with all other SPICE datasets.

Comment (iakov-aws) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:77]: Not only KPI

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md [hunk 4]
```diff
@@ -0,0 +1,1233 @@
...
+    ------------ | -------------
+
+{{% /expand%}}
+
+{{%expand "Click here to continue with the CloudFormation Deployment for the KPI Dashboard" %}}
+
+**NOTE:** An IAM role will be created when you create the CloudFormation stack. Please review the CloudFormation template with your security team and switch to the manual setup if required
+    ------------ | -------------
+
+### Create the KPI Dashboard using a CloudFormation Template
```
Comment (iakov-aws) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:191]: Today users cannot do both with Cloud Formation :(

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md [hunk 5]
```diff
@@ -0,0 +1,1233 @@
...
+
+1. Login via SSO in your Cost Optimization account
```
Comment (iakov-aws) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:193]: Drop SSO

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1_prerequistes.md (hunk 1 above)
Comment (aaronedell) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/1_prerequistes.md:512]: I think we should leave it for now as a stop gap until 655 gets merged. 655 will need some changes to adopt new navigation format.

**Comment by aaronedell** [comment]
I made the changes and pushed to origin. I'm not very good with GitHub so I don't know if that worked.

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md [hunk 6]
```diff
@@ -0,0 +1,1235 @@
...
+### Update Dashboard Template - Optional
+
+{{%expand "Click here to update your dashboard with the latest version" %}}
+
+If you are tracking our [Changelog](https://github.com/aws-samples/aws-cudos-framework-deployment/blob/main/changes/CHANGELOG-kpi.md), you already know that we are always improving the Cloud Intelligence Dashboards.
+
+To pull the latest version of the dashboard from the public template please use the following steps.
+
+#### Option 1: Command Line Tool
+Visit the [GitHub repository](https://github.com/aws-samples/aws-cudos-framework-deployment/) to download and install the CID Command Line Tool and follow the instructions for running the `update` command.
```
Comment (yprikhodko) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:1162]: maybe let's put `cid-cmd update` command to be more clear?

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md [hunk 7]
```diff
@@ -0,0 +1,1235 @@
...
+**NOTE:** The update commands were successfully tested in AWS CloudShell (recommended)
+    ------------ | -------------
+	
+{{% /expand%}}
+
+### Next Steps ###
+- Visit our [customizations lab](https://wellarchitectedlabs.com/cost/200_labs/200_cloud_intelligence/customizations/) to learn some ways to customize your dashboards. 
+- Having trouble? Visit [our FAQ](https://wellarchitectedlabs.com/cost/200_labs/200_cloud_intelligence/faq/). 
+
+{{< prev_next_button link_prev_url="../1_prerequistes/" link_next_url="../2b_cudos_dashboard/" />}}
```
Comment (yprikhodko) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/2_deploy_dashboards.md:1235]: Please fix url to next step

File: content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md [hunk 8]
```diff
@@ -142,6 +142,34 @@ These actions should be done in Governance account
 
 {{% /expand%}}
 
+## I'm getting an error in QuickSight that is saying Athena timed out?
+
+For very large CUR files, Athena may time out trying to query the data for summary_view. In Athena, find the summary_view view, click the three dots next to it and select show/edit query. Modify the following:
+
+- Request an Athena rate limit increase.
```
Comment (yprikhodko) [inline content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md:149]: maybe let's re-phrase 'Request Athena DML query timeout increase via support case' and add link to https://docs.amazonaws.cn/en_us/athena/latest/ug/service-limits.html?

**Review by yprikhodko — APPROVED** [review]

**Comment by aaronedell** [comment]
Changes have been made in the most recent commit. Please merge. Then, less discuss how 655 can be incorporated?

**Pull request merged**